tests/test_grpc.rb

```diff
@@ -22,6 +22,17 @@
 class TestGRPC < TestCase
   @@client = nil # rubocop:disable Style/ClassVars
 
+  def self.runnable_methods
+    # If we're pointed to an NXAPI node (as evidenced by lack of a port num)
+    # then these tests don't apply
+    return [:all_skipped] unless address[/:/]
+    super
+  end
+
+  def all_skipped
+    skip 'Node under test appears to be NXAPI, not gRPC'
```

Comment:
Nit:  Re-word this line to say `Node under test does not appear to use the gRPC client` in case we add other clients besides nxapi and grpc